components/Video/StoryVideo.tsx

```diff
@@ -101,14 +73,16 @@ export const StoryVideo = ({
           ref={videoRef}
           webmSrc={videoWebm}
           mp4Src={videoMp4}
-          onPlay={() => setIsPlaying(true)}
-          onPause={() => setIsPlaying(false)}
+          onPlay={onPlay}
+          onPause={onPause}
           posterSrc={videoPosterSrc}
           className={styles.video}
         />
         <VideoButton
           isPause={isPlaying}
           onClick={toggleVideo}
+          // Use aria-disabled instead of disabled so tab order is preserved
+          aria-disabled={!isVideoInView}
```

Comment:
If the video goes out of viewport, aria-disabled the video button. Instead of using the HTML attribute disabled, I use aria-disabled because it's understood by screen readers, but keeps the button focusable, so focus isn't suddenly lost when the video goes out of view.